cmake/OpenCVDetectInferenceEngine.cmake

```
@@ -134,10 +134,25 @@ endif()
 # Add more features to the target
 
 if(INF_ENGINE_TARGET)
+  if(InferenceEngine_VERSION VERSION_GREATER_EQUAL "2021.4")
+    set(INF_ENGINE_RELEASE "${InferenceEngine_VERSION_MAJOR}")
```

Just curious, would `math` work here?
```suggestion
    math(EXPR INF_ENGINE_RELEASE "${InferenceEngine_VERSION_MAJOR} * 1000000 + ${InferenceEngine_VERSION_MINOR} * 10000 + ${InferenceEngine_VERSION_PATCH} * 100")
```